Comp2022/src/main/include/subsystems/Climber.h

```diff
@@ -88,14 +90,21 @@ class Climber : public frc2::SubsystemBase
 
     // END AUTOGENERATED CODE, SOURCE=ROBOTBUILDER ID=CONSTANTS
 
+    enum

```

Comment:
@connorworley would know better than me, but I think we should make this a named enum type so that we can be more specific in our MoveClimberDistanceInit

```
enum ClimberState {
    FOO = -1,
   BAR = 0,
   BAX = 1.
}
```